lib/runner.js

```diff
@@ -732,6 +732,9 @@ Runner.prototype.runSuite = function(suite, fn) {
 
   debug('run suite %s', suite.fullTitle());
 
+  if (!total && this.forbidEmptySuite) {
+    this.fail(suite, new Error('Empty suite forbidden'));
```

Comment:
`Suite` does not extend `Runnable` (as does `Test` and `Hook`), so you can't `fail` a `Suite`.
The reporter output is weird, if the test file contains only an empty suite.
Maybe a dummy test?